File: agent/src/main/scala/fr/cerostudio/loader/CeroLoader.scala
package fr.cerostudio.loader

import org.spongepowered.asm.mixin.Mixins
import java.lang.instrument.Instrumentation

object CeroLoader {

  def premain(args: String, inst: Instrumentation): Unit = {
    System.out.println("[CeroClient] Starting Mixin (agent mode)...")
    Mixins.addConfiguration("mixins.cero.json")
    System.out.println("[CeroClient] Mixin agent initialized !")
  }
}

File: agent/src/main/java/fr/cerostudio/loader/CeroLoader.java
package fr.cerostudio.loader;

import org.spongepowered.asm.mixin.Mixins;

import java.lang.instrument.Instrumentation;

public class CeroLoader {

    public static void premain(String args, Instrumentation inst) {
        System.out.println("[CeroClient] Starting Mixin...");

        Mixins.addConfiguration("mixins.cero.json");

        System.out.println("[CeroClient] Finished !");
    }
}